dev/README_RELEASE_AIRFLOW.md

```diff
@@ -330,6 +331,37 @@ pipx install -e ./dev/breeze
     --current-release ${VERSION}
     ```
 
+## Publish release candidate documentation (staging)
+
+Documentation is an essential part of the product and should be made available to users.
+In our cases, documentation for the pre-release versions is published in staging S3 bucket.
+The documentation source code and build tools are available in the `apache/airflow` repository, so
+you need to run several workflows to publish the documentation. More details about it can be found in
+[Docs README](../docs/README.md) showing the architecture and workflows including manual workflows for
+emergency cases.
+
+There are two steps to publish the documentation:
+
+1. Publish the documentation to the staging S3 bucket.
+
+The release manager publishes the documentation using GitHub Actions workflow
+[Publish Docs to S3](https://github.com/apache/airflow/actions/workflows/publish-docs-to-s3.yml). You should
+specify the RC tag and choose `s3://staging..` bucket as destination
```

Comment:
name of the bucket seems weird `s3://staging` or `s3://staging...` ? Double dot confuses me, is that 3 dots unfinished ?

Reply:
I was just lazy :)

Reply:
Fixed .